BEIMA.Client/cypress/integration/ABeginningTest.spec.js

```diff
@@ -0,0 +1,18 @@
+/// <reference types="cypress" />
+
+/// The purpose of this test is to login to BEIMA before all the other tests are ran
+/// to ensure that navigating through BEIMA can still happen.
+/// The test database must have a user with this username, but the tester needs to replace the empty
+/// password with a valid one.
+describe("Login to BEIMA", () => {
+  it('Login to BEIMA', () => {
+    cy.visit('http://localhost:3000/')
+    cy.get('[id=username]').scrollIntoView().clear().type("testuser")
+
+    // REPLACE PASSWORD WITH VALID PASSWORD FROM TEST USER IN DATABASE
+    cy.get('[id=password]').scrollIntoView().clear().type("")
+    cy.get('[id=submitBtn]').click()
+
+    cy.wait(700);
```

Comment:
Is there a more reliable way for us to wait that is not just waiting for a specific amount of time for things to load.

Reply:
We can wait for an item to be visible and have a timeout for it, I will play around with that

Reply:
Now that this is in the before block, I think waiting is a bit easier, and when running them in the GUI it still does it really fast,  I'm not sure if it waits the whole 7 seconds is what it feels like